account-settings/set_user_preferences.py

```diff
@@ -14,13 +13,29 @@
 #login_user = raw_input("Enter Mattermost username: ")
 #login_pass = raw_input("Enter Mattermost password: ")
 grouping = raw_input("Enter Sidebar grouping (by_type or none): ")
-grouping = '"'+grouping+'"'
+if grouping == "by_type" or grouping == "none":
```

Comment:
This is perhaps beyond the scope of this script, but I wonder if it would make sense to adopt https://pypi.org/project/argparse-prompt/. All parameters become arguments to the script -- and thus can be invoked without prompts if fully specified -- but anything missing is prompted appropriately.

Reply:
Nice one! Will change it in future versions. Just trying to avoid imports.